Makefile creates .op folder with wrong permissions

### What does it concern?

The Makefile commands and tooling

### Goal or desired behavior

Everytime when I run make build commands on a new system I get afterwards the error

[ERROR] 2022/12/17 12:54:18 Can't continue. We can't safely access "/home/shyim/.op" because its permissions are too broad. Change its permissions to 700 and try again.


### Current behavior

rm -rf .op
run any make tool/build
and you will have afterwards
[ERROR] 2022/12/17 12:54:18 Can't continue. We can't safely access "/home/shyim/.op" because its permissions are too broad. Change its permissions to 700 and try again.


### Relevant log output

_No response_

### op CLI version

_No response_

### shell-plugins local ref

_No response_

--- a/Makefile
+++ b/Makefile
@@ -28,6 +28,7 @@ $(plugins_dir):
 	mkdir -p $(plugins_dir)
 	chmod 700 $(plugins_dir)
 	chmod 700 ~/.op
+	chmod 700 ~/.op/plugins
 
 %/build: $(plugins_dir) registry beta-notice
 	$(eval plugin := $(firstword $(subst /, ,$@)))

--- a/Makefile
+++ b/Makefile
@@ -27,6 +27,7 @@ validate: registry
 $(plugins_dir):
 	mkdir -p $(plugins_dir)
 	chmod 700 $(plugins_dir)
+	chmod 700 ~/.op
 
 %/build: $(plugins_dir) registry beta-notice
 	$(eval plugin := $(firstword $(subst /, ,$@)))

